scalameta/scalameta/shared/src/test/scala/scala/meta/tests/parsers/TemplateSuite.scala

```diff
@@ -87,6 +87,10 @@ class TemplateSuite extends ParseSuite {
     val Class(Nil, Type.Name("A"), Nil, EmptyCtor(), Template(Nil, Nil, self @ EmptySelf(), Nil)) = templStat("class A { this => }")
   }
 
+  test("class A { _ => }") {
+    val Class(Nil, Type.Name("A"), Nil, EmptyCtor(), Template(Nil, Nil, self @ EmptySelf(), Nil)) = templStat("class A { this => }")
```

Comment:
I'm not a fan of these deconstruction tests since they tend to be difficult to debug when they fail. I think we can do something better to diff two trees in testkit, which we could migrate to one day.